### PR TITLE
i18n(fr) Add french translation for `cosmic.mdx`

### PR DESCRIPTION
**Pull request by McFlyPartages**
#### Description (required)
Add french translation for `cosmic.mdx`

#### Related issues & labels (optional)
- Suggested label: i18n

### CODE REVIEW

File: src/content/docs/fr/guides/cms/cosmic.mdx [hunk 1]
```diff
@@ -0,0 +1,243 @@
...
+import PackageManagerTabs from '~/components/tabs/PackageManagerTabs.astro'
+
+
+
+[Cosmic](https://www.cosmicjs.com/) est un [CMS headless](https://www.cosmicjs.com/headless-cms) qui fournit un tableau de bord d'administration pour gérer le contenu et une API qui peut s'intégrer à une gamme variée d'outils frontend.
+
+## Prérequis
+
+1. **Un projet Astro**- Si vous souhaitez démarrer avec un nouveau projet Astro, lisez le [guide d'installation](/fr/install/auto/). Ce guide suit une version simplifiée du [Theme Astro Headless CMS](https://astro.build/themes/details/cosmic-cms-blog/) avec [Tailwind CSS](https://tailwindcss.com/) pour le style.
+2. **Un compte Cosmic et un Bucket** - [Créez un compte Cosmic gratuit](https://app.cosmicjs.com/signup) si vous n'en avez pas. Après avoir créé votre compte, vous serez invité à créer un nouveau projet vide. Il y a aussi un [Modèle simple de Blog Astro](https://www.cosmicjs.com/marketplace/templates/simple-astro-blog) disponible (c'est le même modèle que le thème Astro Headless CMS) pour importer automatiquement tout le contenu utilisé dans ce guide.
```
Comment (thomasbnt) [inline src/content/docs/fr/guides/cms/cosmic.mdx:17]:
```suggestion
2. **Un compte Cosmic et un Bucket** - [Créez un compte Cosmic gratuitement](https://app.cosmicjs.com/signup) si vous n'en avez pas. Après avoir créé votre compte, vous serez invité à créer un nouveau projet vide. Il y a aussi un [modèle simple de Blog Astro](https://www.cosmicjs.com/marketplace/templates/simple-astro-blog) disponible (c'est le même modèle que le thème Astro Headless CMS) pour importer automatiquement tout le contenu utilisé dans ce guide.
```

**Review by thomasbnt — CHANGES_REQUESTED** [review]
Two little errors, the rest is good!

**Review by thomasbnt — APPROVED** [review]
LGTM!

**Review by Yan-Thomas — APPROVED** [review]
LGTM! Thanks peeps 🙌

**Pull request merged**